app/views/admin/partners/_form.html.erb

```diff
@@ -111,33 +111,40 @@
     </div>
     <hr>
     <div class="row">
-      <div class="col-lg-6 col-md-12">
+      <div class="col-lg-8 col-md-12">
         <h2>Opening times</h2>
-        <div id="opening-times" class="opening-times" data="<%= @partner.opening_times.presence || [] %>">
-
-          <div class="added-days">
-            <added-days :list="addedDays"></added-days>
+        <div data-controller="opening-times" data-opening-times-data-value='<%= @partner.opening_times %>'>
+          <ul class="list-group" data-opening-times-target="list">
+          </ul>
+
+          <div class="mt-3 input-group flex-row align-items-center justify-content-sm-between flex-wrap">
+            <label class="m-0" for="day">Day
```

Comment:
i would rather see this using the rails helpers as that is more inline with the rest of the codebase. labels would be something like `f.label :day` and input would be `f.input :time` see [here](https://guides.rubyonrails.org/form_helpers.html#other-helpers-of-interest). and `f.submit`

Reply:
OK this isn't a hill I want to die on so if it's important to you ill change it but my preference is to keep it as is. My main issue is that the format below doesn't look like the stimulus documentation and I find it easier to come back to things that are well documented.

```
<label class="m-0" for="open">Opening Time
    <input class="ml-1" id="open" type="time" name="" value="00:00" data-opening-times-target="open">
</label>
```
```
<%= f.label :open, "Opening Time", <%= f.time :open, class: "ml-1", value: "00:00", data: {opening_times_target: "open"} %> %>
```
I'll leave the final decision to you.

Reply:
yeah i kinda agree with alfie, rails helpers make sense if we are accessing rails models but we're not, this logic only exists for our lil json object, it implies something that's not really true

Reply:
OK so the markup has remained as is but has been moved into a partial with a comment that explains why it doesn't use the same syntax as the rest of the site. This was the agreed compromise at standup so hopefully that resolves this issue.